### ISSUE TITLE
About the optimisation learning rates

### ISSUE THREAD

**Issue by VConchello** [yzhao062/MetaOD#4]
What was the criterion used to choose the learning rates on _core.py:118-136_?
It looks like it alternates between increasing and decreasing during the iterations?
And at the beginning the learning rate is increasing?

**Comment by yzhao062** [comment]
"In METAOD, we employ two strategies that help stabilize the training. First, we leverage meta-feature based (rather than random) initialization. Second, we use cyclical learning rates that help escape saddle points for better local optima [43]."

[43] L. N. Smith. Cyclical learning rates for training neural networks. In WACV, pages 464–472. IEEE Computer Society, 2017.

we do use this technique for better training :)

**Comment by VConchello** [comment]
Mmmh, I see. Thank you for answering so fast and clear :)
